### PR TITLE
Added homework (Lesson-04-Methods)

### PR DESCRIPTION
**Pull request by JeremyMonarch**

### CODE REVIEW

File: Lesson04.Methods/Lesson04.Methods/Program.cs [hunk 1]
```diff
@@ -116,5 +119,64 @@ static int SumNumbers(int a, int b)
...
+
+            else
+            {
+                Console.WriteLine("\nNot Parity \nSum is: " + sum);
+            }
+
+        }
+        static void MaxMinValue(int x, int y)
+        {
+            Console.WriteLine("Max value is: " + Math.Max(x, y));
```
Comment (sergey-olefir) [inline Lesson04.Methods/Lesson04.Methods/Program.cs:158]: try to follow single responsibility principle. Single method should has one responsibility

File: Lesson04.Methods/Lesson04.Methods/Program.cs [hunk 2]
```diff
@@ -116,5 +119,64 @@ static int SumNumbers(int a, int b)
...
+        static void MaxMinValue(double x, double y, double z, double w)
+        {
+            Console.WriteLine("\nMax value is: " + Math.Max(x, Math.Max(y, Math.Max(z, w))));
+            Console.WriteLine("Min value is: " + Math.Min(x, Math.Min(y, Math.Min(z, w))));
+        }
+
+        static bool TrySumIfOdd(int a, int b, out int sum)
+        {
+            sum = a + b;
+            return sum % 2 == 0;
```
Comment (sergey-olefir) [inline Lesson04.Methods/Lesson04.Methods/Program.cs:190]: You must calculate sum between numbers, but not just the sum of two of them

File: Lesson04.Methods/Lesson04.Methods/Program.cs [hunk 3]
```diff
@@ -116,5 +119,64 @@ static int SumNumbers(int a, int b)
...
+        }
+
+        static void Repeat(string x)
+        {
+            x = String.Concat(Enumerable.Repeat(x, 3));
+            Console.WriteLine("Your triple string: " + x);
```
Comment (sergey-olefir) [inline Lesson04.Methods/Lesson04.Methods/Program.cs]: nice try, but can you write this using a loop?

**Review by sergey-olefir — CHANGES_REQUESTED** [review]

File: Lesson04.Methods/Lesson04.Methods/Program.cs (hunk 3 above)
Comment (JeremyMonarch) [reply Lesson04.Methods/Lesson04.Methods/Program.cs]: Added loop for this method

File: Lesson04.Methods/Lesson04.Methods/Program.cs (hunk 2 above)
Comment (JeremyMonarch) [reply Lesson04.Methods/Lesson04.Methods/Program.cs:190]: Fix this part

File: Lesson04.Methods/Lesson04.Methods/Program.cs (hunk 1 above)
Comment (JeremyMonarch) [reply Lesson04.Methods/Lesson04.Methods/Program.cs:158]: Can you a little bit explain about how correct implement SRP in this part of code?

Comment (sergey-olefir) [reply Lesson04.Methods/Lesson04.Methods/Program.cs:158]: there should not be outputs to console. Just return max value

**Review by sergey-olefir — APPROVED** [review]

**Pull request merged**